docs/tutorial/time.rst

```diff
@@ -4,245 +4,78 @@
-sunpy strives to provide convenient and easy methods to do the simple stuff. Python
-already provides an object for a time or date through `datetime.datetime`.
-However, `datetime.datetime` does not provide support for common time formats used in
-solar physics nor leap seconds. To alleviate this, we use `astropy.time.Time` internally
-which allows us to provide a superior user experience.
+This section of the tutorial walks through representing and working with times in sunpy.
+sunpy makes extensive use of the `astropy.time` module for this task.
 
 .. _parse-time:
 
 Parsing Times
 =============
 
-Solar data is associated with a number of different time formats. sunpy provides a simple
-parsing function which can deal with most every format that a user may encounter. Called
-`sunpy.time.parse_time()`, this function can take a variety of inputs.
-
-Strings
--------
-
-The most commonly used are strings and we support a selection of formats
-which are matched using regrex. We currently support the following style of string formats::
-
-    "2007-05-04T21:08:12.999999"
-    "2007/05/04T21:08:12.999999"
-    "2007-05-04T21:08:12.999Z"
-    "2007-05-04T21:08:12"
-    "2007/05/04T21:08:12"
-    "20070504T210812.999999"
-    "20070504T210812"
-    "2007/05/04 21:08:12"
-    "2007/05/04 21:08"
-    "2007/05/04 21:08:12.999999"
-    "2007-05-04 21:08:12.999999"
-    "2007-05-04 21:08:12"
-    "2007-05-04 21:08"
-    "2007-May-04 21:08:12"
-    "2007-May-04 21:08"
-    "2007-May-04"
-    "2007-05-04"
-    "2007/05/04"
-    "04-May-2007"
-    "04-May-2007 21:08:12.999999"
-    "20070504_210812"
-    "2012:124:21:08:12"
-    "2012:124:21:08:12.999999"
-    "20140101000001"
-    "2016.05.04_21:08:12_TAI"
-
-If we pass some of these strings into `sunpy.time.parse_time()`::
+Solar data is associated with a number of different time formats.
+To handle all these formats, sunpy has :meth:`sunpy.time.parse_time` that accepts a variety of inputs, and returns a consistent `~astropy.time.Time` object.
+You might have come across another way of storing time that's built into Python itself, `datetime.datetime`.
+`~datetime.datetime` does not provide support for common time formats used in solar physics or leap seconds, hence the use of `astropy.time.Time` throughout sunpy.
+
+Here's a few examples of using `~sunpy.time.parse_time` to create `~astropy.time.Time` objects::
 
     >>> from sunpy.time import parse_time
     >>> parse_time('2007-05-04T21:08:12')
     <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
-    >>> parse_time('2007/05/04T21:08:12')
-    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
-    >>> parse_time('20070504T210812')
-    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
     >>> parse_time('2007-May-04 21:08:12')
     <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
```

Comment:
```suggestion
```

Reply:
I do not think we need to show a bunch of different time formats. That is what the how-to and reference are for.